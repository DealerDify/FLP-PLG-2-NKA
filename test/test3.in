A,B,C
a,b,c
C
A->aa
B->bbC
C->cc
C->#