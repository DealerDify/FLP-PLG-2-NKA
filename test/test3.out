1,2,3,4,5,6,7,8
abc
3
5,8,3
1,a,4
4,a,5
2,b,6
6,b,3
3,c,7
7,c,8
